### DISCUSSION TITLE
It should really not notify every single time I turn on my computer that there is a new update

### DISCUSSION THREAD

**Discussion by sktvete** [WindowTop/WindowTop-App#312] [Ideas]
Everytime I turn my computer on WindowsTop recommends me updating

**Comment by gileli121** [comment]
Hi, 
Sounds like a bug.
Do you get it every three days less than every three days?
Usually, a new version is released after a few months. This is not something I do a lot because, after this long development, the app is relatively stable and works well. So if there is an update, usually it is after a long time.
